src/pages/index.js

```diff
@@ -71,12 +74,24 @@ export default function WRMCWebsite() {
     setCurrentPage("Show Details");
   }
 
+  // determines the current and next three shows
+  const now = moment();
+
+  const upcomingShows = shows.filter(
+    (show) => (dayToInt[show.time.day] === now.day()) && (show.time.hour >= (now.hour() * 100)));
+
+  upcomingShows.sort((a, b) => compareTwoShows(a, b));
```

Comment:
I think we should create a function that returns the upcoming shows array, this way we can be certain that all this logic will stay in one place.